### PR TITLE
Doc: Clarify request/response attributes are http-only

### PR DESCRIPTION
**Pull request by mandarjog**
Signed-off-by: Mandar U Jog <mjog@google.com>

### CODE REVIEW

**Comment by mandarjog** [comment]
@kyessenov @PiotrSikora

File: docs/root/intro/arch_overview/advanced/attributes.rst [hunk 1]
```diff
@@ -40,7 +40,9 @@ Request attributes
 ------------------
 
 The following request attributes are generally available upon initial request
-processing, which makes them suitable for RBAC policies:
+processing, which makes them suitable for RBAC policies.
+
+`request.*` attributes are only available in http filters.
```
Comment (phlax) [inline docs/root/intro/arch_overview/advanced/attributes.rst:45]: could you use double backticks for literals - ie ` ``request.*`` `

**Review by phlax — CHANGES_REQUESTED** [review]
@mandarjog thanks for this - a couple of rst formatting nits

File: docs/root/intro/arch_overview/advanced/attributes.rst [hunk 2]
```diff
@@ -76,6 +78,8 @@ Response attributes
 
 Response attributes are only available after the request completes.
 
+`response.*` attributes are only available in http filters.
```
Comment (phlax) [inline docs/root/intro/arch_overview/advanced/attributes.rst:81]: same here

**Review by phlax — APPROVED** [review]
lgtm, thanks @mandarjog

**Pull request merged**